include/envoy/upstream/outlier_detection.h

```diff
@@ -48,6 +48,19 @@ class DetectorHostSink {
    * @return the last time this host was unejected, if the host has been unejected previously.
    */
   virtual const Optional<SystemTime>& lastUnejectionTime() PURE;
+
+  /**
+   * @return the success rate of the host in the last calculated interval, in the range -1-100.
```

Comment:
-1-100 is a bit misleading, as it's [0;100] with -1 as a separate value. I like the way you cover this in docs, maybe do the same here.